test/public/envs/overview.test.js

```diff
@@ -16,23 +16,34 @@ const {
     defaultBefore,
     defaultAfter,
     pressElement,
-    getFirstRow,
     goToPage,
     checkColumnBalloon,
     checkEnvironmentStatusColor,
+    validateTableData,
+    expectInnerText,
 } = require('../defaults');
 const { waitForNetworkIdleAndRedraw, waitForTimeout } = require('../defaults.js');
+const dateAndTime = require('date-and-time');
 const { resetDatabaseContent } = require('../../utilities/resetDatabaseContent.js');
 
 const { expect } = chai;
 
+const statusToAcronym = {
+    STANDBY: 'S',
+    DEPLOYED: 'D',
+    CONFIGURED: 'C',
+    RUNNING: 'R',
+    ERROR: 'E',
+    DESTROYED: 'X',
+};
```

Comment:
Dupplicate of `STATUS_ACRONYMS` in lib/public/domain/enums/statusAcronyms.js
BTW please rename the content of the mentioned file to match the conventions
(use await import() to load it asynchronously, making the export async if needed)

Reply:
applied